### PR TITLE
Fix compose features

### PR DESCRIPTION
**Pull request by misstick**
- [x] Fetching after save create a conflict : 2 dispatch on the same Store
 - [x] `ComposeInHTML` options was disabled becauseof bugs
 - [x] Disable `ComposeInHTML` because it is unstable
 - [x] remove buggy focus
 - [x] improve `@hasChanged`
 - [x] fetch after save into `messageACtionCreator`
 - [x] when I send a message, its `body` is empty

### CODE REVIEW

**Comment by frankrousseau** [comment]
@m4dz Could you review that one?

File: client/app/components/compose.coffee [hunk 1]
```diff
@@ -113,109 +108,33 @@ module.exports = Compose = React.createClass
...
 
-        # Each state:change do not send data to server
-        # update date for client modifications
-        @state.lastUpdate = new Date().toISOString()
 
     hasChanged: (props, state) ->
-        @state.lastUpdate isnt @state.date
+        _diff = _.filter @state, (value, key) =>
+            @_oldState[key] isnt value
+        _diff?.length
```
Comment (m4dz) [inline client/app/components/compose.coffee]: I think we can use `_.isEqual` to test changes, nope?

Comment (misstick) [inline client/app/components/compose.coffee]: It doesnt send the same result :(

**Comment by m4dz** [comment]
Thanks! I made another PR to disable the compose module :wink:.

**Pull request merged**